src/App.js

```diff
@@ -1,20 +1,25 @@
 import { useEffect, useState } from "react";
-import './App.css';
-import logo from './mlh-prep.png'
+import "./App.css";
+import SearchBar from "./components/SearchBar/SearchBar";
+import logo from "./mlh-prep.png";
+import {REACT_APP_API_KEY} from './example.js'
 
 function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
-  const [city, setCity] = useState("New York City")
+  const [city, setCity] = useState(null);
   const [results, setResults] = useState(null);
 
   useEffect(() => {
-    fetch("https://api.openweathermap.org/data/2.5/weather?q=" + city + "&units=metric" + "&appid=" + process.env.REACT_APP_APIKEY)
-      .then(res => res.json())
+    console.log(REACT_APP_API_KEY)
+    fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${REACT_APP_API_KEY}`
```

Comment:
This may work locally, but this isn't standard practice when working with environment variables. All environment variables should be stored in a `.env` file so the API keys are not exposed publicly, and they should be accessed through `process.env.NAME_OF_API_KEY`. When you deploy an application, you will need to set your environment variables on the deployed site and the hosting platform will expect the above format.

Reply:
Noted. Thanks. I have reverted the previous commit. I think this should be fixed now.